assets/schemas/manifest.json

```diff
@@ -1481,6 +1518,12 @@
               "jointProcessor",
               "coController"
             ]
+          },
+          "collections": {
```

Comment:
what is this?

Reply:
purposeTemplateCollections - rename?

Reply:
Just asking. Because it looks like an empty object as defined.

Reply:
list of purposeTemplateCollections codes